client/src/pages/Login.jsx

```diff
@@ -21,8 +22,15 @@ function Login() {
       : setPasswordError("Password must be > 6 characters.");
     if (!emailError.length && !passwordError.length) {
       axios
-        .get(`http://localhost:3001/login/${email}/${password}`)
-        .then((data) => console.log(data.data[0]))
+        .post("http://localhost:3001/login/", {
```

Comment:
This is probably your CORS issue, try just doing "/login" or window.location.origin + "/login"